### PR TITLE
Portfolio Desktop version

### PR DESCRIPTION
**Pull request by daudi13**
### In this project I:

- Added the desktop responsive web design
- Used Media query to make sure my website displayed correctly in desktop screen
- Adapted the Figma design
- Used Github flows correctly
- Updated my README.md file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,9 +20,17 @@
     <!-- HEADER SECTION -->
     <header class="header">
         <a href="#" class="header__logo">David</a>
-        <nav class="header__menu-btn">
+        <nav class="header__menu-btn mobile">
```
Comment (Meltrust) [inline index.html:23]: - [x] This main first section should span the whole screen vertically, and have a curvature on the left corner (you could use 'border-radius'):  

Your Implementation | Figma Design
:---:|:---:
image | image

Kindly please correct these, so your main landing screen is striking! 💪

File: index.html [hunk 2]
```diff
@@ -46,7 +54,8 @@ <h2 class="headline-section__sub-title">
 		<section class="work-section">
 			<div class="work-section__box">
 			<div class="work-section__card">
-				<div class="work-section__card--img"><img src="/images/SnapshootPortfolio.png" alt=""></div>
+				<div class="work-section__card--img mobile"><img src="/images/SnapshootPortfolio.png" alt="card snap"></div>
+				<div class="work-section__card--img desktop"><img src="./images/Card-images/SnapshootPortfolioDesktop1.png" alt="card snap"></div>
 				<div class="work-section__card--info">
```
Comment (Meltrust) [inline index.html:59]: - [x] Here, kindly please correct the font styling.  Especially the font-weight and the font-family for these elements on all these sections:

image

That way, your project will look a lot more like the Figma 💪

File: index.html [hunk 3]
```diff
@@ -46,7 +54,8 @@ <h2 class="headline-section__sub-title">
 		<section class="work-section">
 			<div class="work-section__box">
 			<div class="work-section__card">
-				<div class="work-section__card--img"><img src="/images/SnapshootPortfolio.png" alt=""></div>
+				<div class="work-section__card--img mobile"><img src="/images/SnapshootPortfolio.png" alt="card snap"></div>
+				<div class="work-section__card--img desktop"><img src="./images/Card-images/SnapshootPortfolioDesktop1.png" alt="card snap"></div>
```
Comment (Meltrust) [inline index.html:58]: - [x] Kindly please correct these elements so their alignment matches the description and the project title, like in the Figma:

image

- [x] Please kindly add the project titles that are missing too.

Let's do this so the projects featured in your portfolio look more striking! 💪

File: index.html [hunk 4]
```diff
@@ -112,18 +131,23 @@ <h3 class="project-name">Tonic</h3>
...
+						<p class="range__txt mobile">Back End Dev</p>
+						<p class="range__txt desktop">Lead Dev</p>
 						<div class="range__circle"></div>
-						<p class="range__txt">2015</p>
+						<p class="range__txt mobile">2015</p>
+						<p class="range__txt desktop">2018</p>
 					</div>
 					<p class="general__desc">
-						A daily selection of privately personalized reads; no accounts or sign-ups required.
+						A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.
```
Comment (Meltrust) [inline index.html:150]: - [x] In the "about myself" section, there are multiple font styles that do not match the Figma's.  Be especially mindful of the font-family, font-weight, font-size, line-height.   Kindly please correct them.
- [x] There should be a semi-circle in the upper-right corner of the background.  Kindly please add it.

Look:
image

That way, the about section will be a lot more consistent! 💯

Comment (Meltrust) [inline index.html:150]: - [x] Please, kindly correct these lines, so they look like the template (you can use padding, etc):

Your Implementation | Figma Design
:---:|:---:
image | image

Let's do this so your project becomes more striking! 👍

Comment (Meltrust) [inline index.html:150]: - [ ] These background decorations are missing in the "Contact me" section:

image

Please kindly add them, and your portfolio will shine! 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @daudi13, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ShoiraTa** [comment]
### **STATUS: APPROVED  ✔️** 
------
Hi team ,  

Your project is complete!  Congratulations!  🎉💯🌟 
Great job on implementing the requested changes!

## Good Points 👍 
- Project is set up correctly ✔️ 
- Descriptive Pull Request ✔️ 
- Professional documentation ✔️ 



- [ ] OPTIONAL : Did you know that on Figma you are provided with buttons design 😉? I kindly suggest you to add hover effect as provided in Figma:
image


Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**